src/NzbDrone.Core/Indexers/Torznab/Torznab.cs

```diff
@@ -42,6 +42,7 @@ public override IEnumerable<ProviderDefinition> DefaultDefinitions
                 yield return GetDefinition("HD4Free.xyz", GetSettings("http://hd4free.xyz"));
                 yield return GetDefinition("AnimeTosho Torrents", GetSettings("https://animetosho.org", apiPath: @"/feed/nabapi", categories: new int[0], animeCategories: new[] { 5070 }));
                 yield return GetDefinition("Nyaa Pantsu", GetSettings("https://nyaa.pantsu.cat", apiPath: @"/feed/torznab", categories: new int[0], animeCategories: new[] { 5070 }));
+                yield return GetDefinition("Jackett", GetSettings("http://localhost:9117/torznab/YOURINDEXER", categories: new int[] { 5000, 5020, 5030, 5040, 5045, 5050, 5080 }, animeCategories: new[0]));
```

Comment:
Is the aggregate torznab URL similar to this URL? If so should we use that instead, so when someone inevitably doesn't change the URL they get something that works.

Reply:
What would this do? Any documentation regarding this setting?

Reply:
What do you mean? What setting?

Reply:
When adding a new torznab indexer, I get presented 2 options "custom" and "jackett". How do I use the Jackett option?. With custom I copy the link from jackett "torznab" button without issues, but what does the "jackett" option do? "http://localhost:9117/torznab/YOURINDEXER" What is "YOURINDEXER"?

Reply:
`YOURINDEXER` would need to be replaced with the matching portion from the actual Jackett URL.